pipelines.adoc

```diff
@@ -9,21 +9,47 @@ permalink: /guides/pipelines-getting-started/
+```
+
+When the Kabanero operator activates the CRD, it will associate the pipelines in the pipelines archive with each of the stacks in the stack hub.  The default pipelines are intended to work with all the stacks in the stack hub in the example above. All of the pipeline related resources like the tasks, trigger bindings, and pipelines prefix the name of the resource with the keyword `StackId`.  When the operator activates these, it will replace the keyword with the name of the stack it is activating.
+
+=== Updating Default Tasks & Pipelines
+
+The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive to be used by Kabanero CRD is to drive the  link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh[package.sh] script.  The script will generate the archive file with the necessary pipeline artifacts and a manifest.yaml describing the contents of the archive.  
+
+=== Creating New Tasks & Pipelines
 
-If you are building a new application stack, the default tasks and pipelines are automatically pulled into your application stack hub repo. If you want to customize the default tasks or pipelines, you can choose to apply your changes either to all application stacks or to a specific application stack.  To apply your changes to all,  update the files in the `incubator/common/pipelines/default` directory in your application stack hub repo. If you want to update tasks or pipelines for one, make your changes in the pipelines directory under that application stack in your application stack hub repo.
 
 === The build, push and deploy pipeline
 
-https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-deploy-pl.yaml[build-deploy-pl.yaml]
+- link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/build-deploy-pl.yaml[build-deploy-pl.yaml]
 
-This file is the primary pipeline that is associated with the application stack hub. It validates that the application stack is active, builds the Git source, publishes the container image, and deploys the application. It looks for `git-source` and `docker-image` resources that are used by the `build-task` and `deploy-task` files.
+This file is the primary pipeline that showcases all the tasks supplied in the Kabanero repo. It validates that the application stack is active, builds the application stack, publishes the application image to the container registry, does a security scan of the image, and conditionally deploys the application. When running the pipeline via a webhook, the pipeline  leverages the tekton triggers functionality to conditionally deploy the application only when a pull request is merged in the git repo.  Other actions that trigger the pipeline run, will validate, build, push, and scan the image.
```

Comment:
Change
the pipeline  leverages the tekton triggers functionality to 
to
the pipeline leverages triggers to

Is "run" in the final sentence a noun or a verb?  Depending,
Change
Other actions that trigger the pipeline run, will validate,
to (if a verb) 
Other actions that trigger the pipeline run, validate,
or (if a noun) 
Other actions that trigger the pipeline run validate,